test/e2e/operator_test.go

```diff
@@ -1192,3 +1260,21 @@ func upstreamTLSPod(name, ns, image string, configMap *corev1.ConfigMap) *corev1
 		},
 	}
 }
+
+func getClusterDNSOperatorPods(cl client.Client) (*corev1.PodList, error) {
```

Comment:
```suggestion
func getClusterDNSOperatorPods(cl client.Client) (*corev1.PodList, error) {
	t.Helper()
```

Reply:
Add t.Helper() here as well as other helper functions that should not be treated as standalone tests.

Reply:
I've addressed this by moving the helper functions out of the `operator_test.go` file.